src/compiler/types.rs

```diff
@@ -1252,6 +1321,26 @@ pub enum Ternary {
     True = -1,
 }
 
+impl TryFrom<i32> for Ternary {
```

Comment:
Per [this answer](https://stackoverflow.com/a/57578431/732366), convert `i32` -> enum variant (used below in the implementation of `BitAndAssign`)